cmd/clusterctl/client/cluster/mover_test.go

```diff
@@ -788,3 +788,36 @@ func Test_objectsMoverService_checkTargetProviders(t *testing.T) {
 		})
 	}
 }
+
+func Test_objectsMoverService_ensureNamespaces(t *testing.T) {
+	for _, tt := range moveTests {
```

Comment:
I don't think it is required to run this for all the move Tests, because move test are all in the same namespace (differences are not relevant for this test)

Reply:
What about instead having a single, simple list of objects with
- a cluster object in namespace-1 (that is a namespace that already exists in the target)
- a cluster object in namespace-2 (that is a namespace that does not exist exists in the target)

Reply:
> a namespace that already exists in the target

you have to create it using toProxy

Reply:
Thank you for these suggestions. Yes I think it would be worth creating a new list with objects in these namespaces